packages/components/src/internal/components/editable/utils.test.ts

```diff
@@ -792,8 +798,11 @@ describe('getValidatedEditableGridValue', () => {
             });
         });
 
-        validValues = [1.11, '100', '1:00 AM', '1:00 PM', '13:24'];
-        results = [' 01:11', ' 10:00', ' 01:00', ' 13:00', ' 13:24'];
+        // TODO: Why are we considering the numbers like 1.11 and 100 a valid time values?
```

Comment:
only because moment seemed to allow it. If those can now be considered invalid, that is even better.